server/src/main/webapp/WEB-INF/rails/webpack/views/pages/new-environments/environments_widget.tsx

```diff
@@ -79,12 +79,18 @@ export class EnvironmentWidget extends MithrilViewComponent<EnvAttrs> {
         </div>
       </div>;
     }
+    let deleteTitle;
+    if (!environment.canAdminister()) {
+      deleteTitle = `You are not authorized to delete the '${environment.name()}' environment.`;
+    } else if (!environment.isLocal()) {
+      deleteTitle = `Cannot delete '${environment.name()}' environment as it is partially defined in config repo.`;
```

Comment:
can we change `Cannot delete '${environment.name()}' environment as it is partially defined in config repository.`?

Changing `repo` => `repository`?

Reply:
will change it.